department-of-veterans-affairs/caseflow
# Judge assign task view performance

References #8711

### Description
Navigating to the judge's assign view does not make a request for the individual judge team attorney tasks until the user selects that attorney

### Acceptance Criteria
- [x] Task counts are still correct in the sidebar
- [x] Attorney tasks are only loaded after clicking into their name
- [x] Task counts are still correct in the sidebar after assigning a task

### Testing Plan
1. Go to Judge assign view for BVAAABSHIRE
2. Ensure no calls were made for attorney tasks (will appear as `<attorney_id>?role=judge` and `tasks?user_id=<attorney_id>&role=attorney`)
3. Ensure the number of tasks is correct in the sidebar
4. Assign a task to an attorney
3. Ensure the number of tasks updates in the sidebar
6. Click on one of the attorney's names to ensure those requests are now made and the attorney's tasks appear
7. Assign another attorney a task from an attorney page to make sure the number of tasks in the sidebar is accurate
